src/Sylius/Bundle/InstallerBundle/Command/SetupCommand.php

```diff
@@ -108,7 +108,7 @@ protected function setupAdministratorUser(InputInterface $input, OutputInterface
             } while ($exists);
 
             $user->setEmail($email);
-            $user->setPlainPassword($this->askHidden($output, 'Choose password:', array(new NotBlank())));
+            $user->setPlainPassword($this->getAdministratorPassword($output));
```

Comment:
This should be `$dialog->askHiddenResponseAndValidate()`, where `$dialog` is default Symfony helper.